cli/cmd/login.go

```diff
@@ -129,21 +130,37 @@ func runLogin(ctx context.Context, cmd *cobra.Command) (err error) {
 
 	url := accountsURL + "?returnTo=" + localServerURL + "/callback"
 	if err := browser.OpenURL(url); err != nil {
-		fmt.Printf("Failed to open browser at %s. Please open the URL manually.\n", accountsURL)
+		fmt.Printf("Failed to open browser. Please open %s manually and paste the token below:\n", accountsURL)
+		oldState, err := term.MakeRaw(int(os.Stdin.Fd()))
+		if err != nil {
+			return fmt.Errorf("failed setting stdin to raw mode: %w", err)
+		}
+		tty := term.NewTerminal(os.Stdin, "")
+		refreshToken, err = tty.ReadLine()
+		term.Restore(int(os.Stdin.Fd()), oldState)
+
+		if err != nil {
+			return fmt.Errorf("failed to read token: %w", err)
+		}
+
+		refreshToken = strings.TrimSpace(refreshToken)
 	} else {
 		fmt.Printf("Opened browser at %s. Waiting for authentication to complete.\n", url)
-	}
 
-	// Wait for an OS signal to begin shutting down.
-	select {
-	case <-ctx.Done():
-		fmt.Println("Context cancelled. Shutting down server.")
-	case <-gotToken:
+		// Wait for an OS signal to begin shutting down.
+		select {
+		case <-ctx.Done():
+			fmt.Println("Context cancelled. Shutting down server.")
+		case <-gotToken:
+		}
 	}
 
 	if refreshToken == "" {
 		return fmt.Errorf("failed to get refresh token")
 	}
+
+	fmt.Println("Authenticating...")
```

Comment:
Just thought some kind of feedback after the input/click is complete, before we hit the server would be nice...